src/cf-forms/src/atoms/text-input.less

```diff
@@ -6,7 +6,7 @@
 
     display: inline-block;
 
-    padding: unit( 6px / @base-font-size-px, em );
+    padding: unit( 7px / @base-font-size-px, em );
```

Comment:
I figured it was the same all around before it should be after as well. If not, should left/right be 5px?